about/fair-initiatives.md

```diff
@@ -0,0 +1,46 @@
+
+The FAIR Web Foundation is currently working on formalizing a governing board. Once formed, the Foundation will exercise its duties according to the charter it has adopted.
+
+The Technical Steering Committee of the FAIR Package Manager is solely responsible for determining technical direction, governance, and quality of the project. The leadership of the FAIR Package Manager is a triumvirate of three equal co-chairs serving three year terms, rotating annually.
+
+## **Supply Chain Security**
+
+Work is underway to identify and analyze specific risks and threats to the software supply chain, with a view to verifying or enhancing mitigation strategies set in place with the FAIR Protocol.
+
+## **FAIR Protocol**
+
+The initial draft of the FAIR Protocol is based on ATProto and the W3C’s DID Specification. The specifications within the protocol are in review with portions in draft status pending any necessary revisions determined through the implementation process, security reviews, and trust moderation requirements. FAIR recognizes the significant work by Ryan McCue in envisioning and drafting the initial protocol specification in early 2025 for review, revision, and adoption by the TSC.
+
+[FAIR Protocol on GitHub](https://github.com/fairpm/fair-protocol)
+
+## **Trust Moderation (Labeling)**
+
+Work is currently underway to refine and ratify the labeling specification, which is based on Bluesky’s moderation architecture, Ozone. FAIR envisions operating a single *required* labeling service which will assign trust labels to aid with verification with appropriate levels of trust in the identity and/or authenticity of entities and/or packages. A draft specification exists, with a working group tasked with determining the necessary business rules required to establish levels of trust within the FAIR network, or ecosystem.
+
+[Moderation Spec on GitHub](https://github.com/fairpm/fair-protocol/tree/main/docs/moderation)
+
+## **Software**
+
+Under specific working groups, FAIR is publishing software for use in aggregating and serving software packages, labeling packages with specific trust signals, searching for and installing packages from the federated network, serving packages from a synchronized mirror of the [WordPress.org](http://wordpress.org) repository, and other related packages. These packages consist of WordPress plugins, standalone PHP or other scripting projects, and libraries consisting of reusable PHP classes for inclusion of specific functions within other projects. Not all are WordPress specific. Software releases are licensed under the GPL v.2.0 or later, or under the MIT license. Each software project falls under the auspices of a specific Working Group to publish and maintain the software.
```

Comment:
Why "Under specific working groups"?

Reply:
I suggest making these change requests as new PRs; this particular PR is literally just making the actual published content live, so it doesn't really make a lot of sense to make changes in this context.